staging/src/k8s.io/client-go/discovery/discovery_client_test.go

```diff
@@ -547,8 +547,8 @@ func TestGetOpenAPISchema(t *testing.T) {
 	if err != nil {
 		t.Fatalf("unexpected error getting openapi: %v", err)
 	}
-	if e, a := returnedOpenAPI(), got; !reflect.DeepEqual(e, a) {
-		t.Errorf("expected %v, got %v", e, a)
+	if e, a := returnedOpenAPI(), got; !golangproto.Equal(e, a) {
```

Comment:
why is this needed?

Reply:
Its not legal to do reflect.DeepEqual on protos generated by newer protoc versions. https://groups.google.com/g/golang-nuts/c/-p-KiwYE598 describes this a bit.

The issue is data gets encoded into private fields that are implementation details and should not be directly compared. The recommended way is actually using protocmp but I didn't want to introduce a new dependency

Reply:
ok. why adding '\n' in the line below?

Reply:
oh that was to make it easier to view the failure, I can revert if necessary

Reply:
that is fine as is. just wondering that '\n' is only added in this one line below, but not in others.

Reply:
> Its not legal to do reflect.DeepEqual on protos generated by newer protoc versions. 

I see changes to paths the might be hit by openapi serving code, which cares about equality (https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiextensions-apiserver/pkg/controller/openapi/controller.go#L211-L213)

@sttts, do we have tests in place that would catch spurious detection of deltas